openhtf/output/web_gui/__init__.py

```diff
@@ -368,6 +368,8 @@ def handle_test_state_update(self, hostport, test_uid, state):
     This handler updates our URL handlers with any newly available plug
     information and notifies the StationPubSub of the event.
     """
+    if state is None:
+      return
```

Comment:
I feel like this would be better with a blank line between `return` and  `_make_sockjs_url`, but YMMV (the [style guide](https://google.github.io/styleguide/pyguide.html#Blank_Lines) leaves it up to personal judgement).